src/app/globals.css

```diff
@@ -365,6 +365,9 @@ input[type="number"] {
     .gridViewMain-4 {
         @apply grid grid-cols-4;
     }
+    .gridViewMain-4-5 {
+        @apply grid grid-cols-4 grid-rows-5;
+    }
```

Comment:
https://tailwindcss.com/docs/reusing-styles#avoiding-premature-abstraction

From the tailwinds guide

> Whatever you do, don’t use @apply just to make things look “cleaner”. Yes, HTML templates littered with Tailwind classes are kind of ugly. Making changes in a project that has tons of custom CSS is worse.

> If you start using @apply for everything, you are basically just writing CSS again and throwing away all of the workflow and maintainability advantages Tailwind gives you, for example: